src/olympia/constants/webext_permissions.py

```diff
@@ -16,6 +16,7 @@
     u'http://*/*': ALL_URLS_PERMISSION,
     u'https://*/*': ALL_URLS_PERMISSION,
     u'*://*/*': ALL_URLS_PERMISSION,
+    # FIXME: how about https://*/path ?
```

Comment:
Sounds good. IMHO you don't even need the final `*`, ultimately if the extension affects every host it's just as dangerous regardless of what paths are concerned. 

Do you want me to remove the `FIXME` to make it easier to merge later ?